packages/@core.keystone/test.utils.js

```diff
@@ -97,151 +114,194 @@ const prepareNextExpressApp = async (dir) => {
-                    }
-                    return resolve(body)
-                })
-            })
+        includeExtensions: true,
+        isExtractableFile: (value) => {
+            return value instanceof UploadingFile
         },
-        query: async (query, variables = {}) => {
-            if (query.kind !== 'Document') throw new Error('query is not a gql object')
-            return new Promise((resolve, reject) => {
-                setupSupertest(client.get(API_PATH)).set('Cookie', [cookiesToString(cookies)]).query({
-                    query: print(query),
-                    variables: JSON.stringify(variables),
-                }).end(function (err, res) {
-                    if (err) {
-                        console.error(err)
-                        return reject(err)
-                    }
-                    const setCookies = res.headers['set-cookie']
-                    if (setCookies) {
-                        cookies = { ...cookies, ...extractCookies(setCookies) }
-                    }
-                    const body = res.body
-                    if (body && body.errors && TESTS_LOG_FAKE_CLIENT_RESPONSE_ERRORS) {
-                        console.warn(util.inspect(body.errors, { showHidden: false, depth: null }))
-                    }
-                    return resolve(body)
-                })
-            })
+        FormData,
+        formDataAppendFile: (form, name, file) => {
+            form.append(name, file.stream)
         },
-    }
-}
+        useGETForQueries: true,
+        fetch: (uri, options) => {
+            options.headers = { ...options.headers, ...customHeaders }
+            if (cookiesObj && Object.keys(cookiesObj).length > 0) {
+                options.headers = { ...options.headers, cookie: restoreCookies() }
+            }
 
-const makeRealClient = async () => {
-    // TODO(pahaz): remove axios! need something else ... may be apollo client
-    const serverUrl = new URL(TESTS_REAL_CLIENT_REMOTE_API_URL).origin
-    const cookieJar = new CookieJar()
-    const client = axios.create({
-        withCredentials: true,
-        adapter: require('axios/lib/adapters/http'),
-        headers: {
-            'Content-Type': 'application/json',
-            Accept: 'application/json',
-            Cache: 'no-cache',
+            return fetch(uri, options)
+                .then((response) => {
+                    const setCookieHeader = response.headers.raw()['set-cookie']
+                    if (setCookieHeader) {
+                        // accumulate cookies received from the server
+                        saveCookies(setCookieHeader)
+                    }
+                    return response
+                })
         },
-        validateStatus: (status) => status >= 200 && status < 500,
+    }))
+
+    const client = new ApolloClient({
+        cache: new InMemoryCache({
+            addTypename: false,
+            resultCaching: false,
+            typePolicies: {
```

Comment:
why do you need this stuff here?! Are you disabling cache and adding the cache policy?

Reply:
I forgot to remove after tests. Fixed.